x-pack/platform/packages/shared/kbn-cloud-security-posture/common/utils/helpers.ts

```diff
@@ -195,3 +201,26 @@ export const buildEntityAlertsQuery = ({
     },
   };
 };
+
+export const setIsNewIndexAliasCreated = (value: boolean) => {
+  IS_INTEGRATION_INCLUDES_TRANSFORM = value;
+};
+
+export const getIsNewIndexAliasCreated = (): boolean => {
```

Comment:
`IsNewIndexAliasCreated` internally checks for `IS_INTEGRATION_INCLUDES_TRANSFORM,` which on one hand is self-explanatory for those who worked on the feature (new transform contains this new alias, that's why we can rely on the transform existence to check if alias has been created). But it might confuse the rest of the folks or even us after some time. Maybe having a comment explaining this logic would be beneficial